composer.json

```diff
@@ -27,7 +27,7 @@
             "/Tests/"
         ]
     },
-    "minimum-stability": "stable",
+    "minimum-stability": "dev",
```

Comment:
@sroze this should be changed back to `stable` once symfony 4.3 is officially released.

Reply:
Symfony 4.3 was released, can we please have this updated and new version tagged?